accelerator-home-ui/src/screens/SplashScreens/LogoScreen.js

```diff
@@ -49,8 +58,14 @@ export default class LogoScreen extends Lightning.Component {
         return 'right'
     }
 
-    _init() {
+    async _init() {
         this.btApi = new BluetoothApi()
+        await appApi.getPluginStatus("org.rdk.Bluetooth")
+            .then(()=> this._isBluetoothExist = true)
+            .catch(()=>this._isBluetoothExist = false)
+        await appApi.getPluginStatus("org.rdk.RemoteControl")
```

Comment:
What will happen when we have only Bluetooth plugin not RemoteControl ?

Reply:
App will navigate to splash/bluetooth [remote pairing] screen.

- When both RemoteControl and Bluetooth plugins are not found, then an error message will be displayed asking user to click okay [to check if user is able to interact with the UI]. if 'Ok' then app will be navigated to language selection screen[splash/language].
- when both plugins or any one of the plugins either remote control or bluetooth is present , then app will follow the normal navigation flow which is pairing screen [splash/bluetooth] after logo screen.